monitoring/configuring-the-monitoring-stack.adoc

```diff
@@ -119,6 +119,22 @@ include::modules/monitoring-setting-the-body-size-limit-for-metrics-scraping.ado
 * link:https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config[Prometheus scrape configuration documentation]
 endif::openshift-dedicated,openshift-rosa[]
 
+// Configuring limits and resource requests for monitoring components
+
+[id="managing-cpu-and-memory-resources-for-monitoring-components"]
+== Managing CPU and memory resources for monitoring components 
+
+You can ensure that the containers that run monitoring components have enough CPU and memory resources by specifying values for resource limits and requests for these components.
```

Comment:
```suggestion
You can ensure that the containers that run monitoring components have enough CPU and memory resources by specifying values for resource limits and requests for those components.
```
It might just be me, but "those components" sounds more natural than "these components" here. I tend to equate "these" with "the following" and "those" as a pronoun referring to something that was already mentioned.

Reply:
Agree. Changed.